### ISSUE TITLE
about section 5.5

### ISSUE THREAD

**Issue by YaoShunyu19** [SaMMyCHoo/Light-VQA-plus#1]
Hi!
In section 5.5, how exactly is LIGHT-VQA+ used to finetune FECNet? Is the output of LIGHT-VQA+ used as an additional loss to optimize FECNet during its training?
Thanks!

**Comment by SaMMyCHoo** [comment]
Thank you very much for your interest in our work! 
You are right. In the experiments of fine-tuning the FEC-Net, the output of Light-VQA+ is added as an additional loss to the original loss while training the FEC-Net.

**Comment by YaoShunyu19** [comment]
Thank you for your reply!
